sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/autorest.md

```diff
@@ -14,10 +14,21 @@ Run `generate.ps1` in this directory to generate the code.
 
 > see <https://aka.ms/autorest>
 
+``` yaml
+# Change accessibility
+directive:
+  from: timeseriesinsights.json
+  where: $.definitions.UpdateModelSettingsRequest
+  transform: >
+    $["x-accessibility"] = "internal"
```

Comment:
This allows to change accessibility in bulk.

Reply:
Unrelated to this PR, but this will be a great addition to the ADT code since a lot of the customizations we do is simply to change accessibility of models to internals. So we can do $.definitions.* to change all models (or be selective). I can make that change in ADT.

Reply:
Totally agree, thanks for this suggestion.